src/test/java/uk/gov/companieshouse/filinghistory/consumer/delta/UpsertDeltaServiceTest.java

```diff
@@ -34,13 +35,16 @@ class UpsertDeltaServiceTest {
     @Mock
     private InternalFilingHistoryApi apiRequest;
 
-    @Test
-    void shouldSuccessfullyPassDeserialisedAndMappedDeltaToApiClient() throws Exception {
+    @ParameterizedTest
+    @CsvSource({
+            "TM01"
+    })
+    void shouldSuccessfullyPassDeserialisedAndMappedDeltaToApiClient(final String prefix) throws Exception {
```

Comment:
This can be a regular unit test as the return from deserialising the string is a Mock anyway. The argument passed to ```new ChsDelta()``` can be any string and does not need to be read from a file - that is tested in the consumer integration test anyway.